docs/getting_started/tutorials/tutorial_lattice.rst

```diff
@@ -0,0 +1,325 @@
+``Lattice`` Tutorial
+====================
+
+The following notebook provides a thorough walkthrough to using the
+``Lattice`` class to build up crystal systems.
+
+``Lattice`` Functionality
+-------------------------
+
+-  **Variable-dimension crystal structures**
+
+   -  ``Lattice`` can support the dimensionality of ``mBuild``, which
+      means that the systems can be in 1D, 2D, or 3D. Replace the
+      necessary vector components with 0 to emulate the dimensionality
+      of interest.
+
+-  **Multicomponent crystals**
+
+   -  ``Lattice`` can support an indefinite amount of lattice points in
+      its data structure.
+   -  The ‘repeat’ cell can be as large as the user defines useful.
+   -  The components that occupy the lattice points are
+      ``mbuild.Compound's``.
+
+      -  This allows the user to build any system since compounds are
+         only a representation of matter in this design.
+      -  Molecular crystals, coarse grained, atomic, even alloy crystal
+         structures are all supported
+
+-  **Triclinic Lattices**
+
+   -  With full support for triclinic lattices, any crystal structure
+      can technically be developed.
+   -  Either the user can provide the lattice parameters, or if they
+      know the vectors that span the unit cell, that can also be
+      provided.
+
+-  *IN PROGRESS* **Generation of lattice structure from crystallographic
+   index file (CIF)**
+
+   -  Although this feature is not currently implemented, this
```

Comment:
You could link here to the `Load files` portion of quick start to show how this could be done in a roundabout route.

Reply:
I added this line with a link as you suggested.

Please also see the :ref:`QuickStart_Load_files` section for other ways to load files.